src/app/configureApp.jsx

```diff
@@ -1,71 +1,70 @@
-import React from 'react';
-import ApplicationMenu from './ApplicationMenu';
+import RoutingMenu from 'terra-application-layout/lib/menu/RoutingMenu';
 import Components from './components/Components';
-import ComponentsMenu from './components/ComponentsMenu';
 import Home from './components/Home';
 
-const injectConfig = configuredProps => (
-  Component => (
-    props => (
-      <Component {...props} {...configuredProps} />
-    )
-  )
-);
-
 const buildComponent = (Component, configuredProps) => (
   {
     default: {
-      componentClass: injectConfig(configuredProps)(Component),
+      componentClass: Component,
+      props: configuredProps,
     },
   }
 );
 
-const buildSubNavigationConfig = (array, config, ComponentMenu, exampleType, pathRoot) => {
-  config.map((componentKey) => {
-    const componentPath = componentKey.path;
-    const examples = componentKey[`${exampleType}`];
-
-    if (componentPath && examples) {
-      const path = pathRoot + componentPath;
-      examples.forEach((example) => {
-        if (example[`${exampleType}`]) {
-          buildSubNavigationConfig(array, examples, ComponentMenu, exampleType, path);
-        }
-      });
-
-      // Do not create a submenu for the component if the component has one site page with no additional sub-nav.
-      if (exampleType !== 'tests' && examples.length === 1 && !examples[0][`${exampleType}`]) {
-        return undefined;
+const buildMenuConfig = (component, menuComponent, exampleType, pathRoot = '') => {
+  let generatedConfig = {};
+  const path = pathRoot + component.path;
+  const examples = component[exampleType];
+
+  if (examples) {
```

Comment:
Does this create links for config where a page has a component but not examples?

-this git diff is hard to follow

Reply:
This both creates and adds sub menus to the flat list of menu routes and adds items to the current menu.